python/cql2.pyi

```diff
@@ -26,6 +26,42 @@ class Expr:
             >>> expr = Expr.from_path("fixtures/text/example01.txt")
         """
 
+    @staticmethod
+    def parse_text(s: str) -> Expr:
```

Comment:
I personally prefer `from_text` and `from_json` for method that return a new instance of the class

Reply:
@vincentsarago should we have an initializer at all? Or should we force folks to go through `parse_file`, `parse_json`, `parse_text`, or `parse`?